src/app/api/webhook/route.ts

```
@@ -0,0 +1,273 @@
+
+    const event = body;
+    console.log(`Received Stream Webhook: ${event.type}`);
+
+    switch (event.type) {
+      case "call.session_started":
+        console.log(`[Webhook] Received call.session_started for CID: ${event.call?.cid || event.call_cid}`);
+        await handleCallSessionStarted(event);
+        break;
+
+      case "call.session_participant_joined":
+        const participantId = event.participant?.user?.id;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)